pkgs/servers/sql/sqlite/jdbc/default.nix

```diff
@@ -0,0 +1,35 @@
+
+
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "3.20.0";
+  pname = "sqlite-jdbc";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://bitbucket.org/xerial/${pname}/downloads/${name}.jar";
+    sha256 = "0wxfxnq2ghiwy2mwz3rljgmy1lciafhrw80lprvqz6iw8l51qfql";
+  };
+
+  unpackPhase = "true";
+
+  buildInputs = [
+  ];
+
+  buildPhase = "true";
```

Comment:
I think that either `dontBuild = true` or `phases = [ "installPhase" ]` is a more suitablesolution

Reply:
Seems like good advice, will do.